plugins/node/opentelemetry-instrumentation-aws-lambda/README.md

```diff
@@ -69,6 +70,18 @@ new AwsLambdaInstrumentation({
 })
 ```
 
+### Specifying the Lambda Handler
+
+The instrumentation will attempt to automatically determine the Lambda handler function to instrument. To do this, it relies on the `_HANDLER` environment variable which is [set by the Lambda runtime](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html#configuration-envvars-runtime). For most use cases, this will accurately represent the handler that should be targeted by this instrumentation.
+
+There exist use cases where the instrumetation's default behavior will not work. For these use cases, the `lambdaHandler` option should be used to explicitly specify the Lambda handler.
+
+The `lambdaHandler` should be specified as a string in the format `<file>.<handler>`, where `<file>` is the name of the file that contains the handler and `<handler>` is the name of the handler function. For example, if the handler is defined in the file `index.js` and the handler function is named `handler`, the `lambdaHandler` should be specified as `index.handler`.
+
+To better explain when `lambdaHandler` should be specified, consider how some telemetry tools, such as [Datadog](https://www.datadoghq.com/), are instrumented into the Lambda runtime. Datadog does this by overriding the handler function with a wrapper function that is loaded via a [Lambda Layer](https://docs.aws.amazon.com/lambda/latest/dg/chapter-layers.html). In these examples, the Lambda's handler will point to the Datadog wrapper and not to the actual handler that should be instrumented. In cases like this, `lambdaHandler` should be used to explicitly specify the handler that should be instrumented.
```

Comment:
Another small nit: Can this paragraph be placed before the previous paragraph? I think it fits better after the sentence that starts with "There exist use cases" since it explains those use cases in more detail here. Once those use cases are established, then the format of the `lambdaHandler` can be outlined

Reply:
Yes, I definitely see what you mean. Just made that adjustment!